net/jitsi-meet-electron/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jitsi jitsi-meet-electron 2023.8.0 v
+github.tarball_from releases
+
+name                jitsi-meet-electron
+version             2023.8.0
```

Comment:
the "name" and "version" fields are automatically set from the "github.setup" line and are this redundant, please remove